src/stories/Library/search-result-item/search-result-item.scss

```diff
@@ -67,6 +67,12 @@
   margin-top: 10px;
 
   @include breakpoint-s {
+    display: -webkit-box;
+    -webkit-line-clamp: 2;
+    -webkit-box-orient: vertical;
     margin-top: 16px;
+    max-width: 407px;
```

Comment:
The value `407px` seems to correlate with the `grid-template-columns` on L14. I wonder if both are necessary.

Also note that Chromatic shows that the availability labels are shifted a bit to the left with this change. I wonder if that is intentional.